From 63948ec3b38334a962e3e4edda9100428c35e381 Mon Sep 17 00:00:00 2001
From: Piotr Karbowski <piotr.karbowski@gmail.com>
Date: Sun, 27 Feb 2011 17:03:41 +0100
Subject: [PATCH] Set initscripts to /bin/bash instead of /bin/sh.

Because gentoo's developers love using bashism in init scripts what ends with not working scripts if /bin/sh is not a bash (or symlink to).

---
 mk/scripts.mk |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/mk/scripts.mk b/mk/scripts.mk
index bd7ec6b..3b5876a 100644
--- a/mk/scripts.mk
+++ b/mk/scripts.mk
@@ -12,7 +12,7 @@ _PKG_SED!=		${_PKG_SED_SH}
 _LCL_SED_SH=		if test "${PREFIX}" = "${LOCAL_PREFIX}"; then echo "-e 's:@LOCAL_PREFIX@::g'"; else echo "-e 's:@LOCAL_PREFIX@:${LOCAL_PREFIX}:g'"; fi
 _LCL_SED!=		${_LCL_SED_SH}
 
-SED_REPLACE=		-e 's:@SHELL@:${SH}:g' -e 's:@LIB@:${LIBNAME}:g' -e 's:@SYSCONFDIR@:${SYSCONFDIR}:g' -e 's:@LIBEXECDIR@:${LIBEXECDIR}:g' -e 's:@PREFIX@:${PREFIX}:g' ${_PKG_SED}$(shell ${_PKG_SED_SH}) ${_LCL_SED}$(shell ${_LCL_SED_SH})
+SED_REPLACE=		-e 's:@SHELL@:/bin/bash:g' -e 's:@LIB@:${LIBNAME}:g' -e 's:@SYSCONFDIR@:${SYSCONFDIR}:g' -e 's:@LIBEXECDIR@:${LIBEXECDIR}:g' -e 's:@PREFIX@:${PREFIX}:g' ${_PKG_SED}$(shell ${_PKG_SED_SH}) ${_LCL_SED}$(shell ${_LCL_SED_SH})
 
 # Tweak our shell scripts
 .SUFFIXES:	.sh.in .in
-- 
1.7.4.1

